pkg/skaffold/docker/image_test.go

```diff
@@ -171,18 +171,64 @@ func TestDigest(t *testing.T) {
 }
 
 func TestGetBuildArgs(t *testing.T) {
-	artifact := &v1alpha2.DockerArtifact{
-		BuildArgs: map[string]*string{
-			"key1": util.StringPtr("value1"),
-			"key2": nil,
+	type args struct {
+		a *v1alpha2.DockerArtifact
+	}
+	tests := []struct {
+		name string
+		args args
+		want []string
+	}{
+		{
+			name: "build args",
+			args: args{
```

Comment:
WDYT of inlining args? Since it contains a single arguments, I think it makes sense.
Also, most tests use `description` instead of `name` to describe the test.